.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       fail-fast: false
       matrix:
         java: ['adopt@1.8', 'adopt@1.11']
-        scala: ['2.11.12', '2.12.12', '2.13.3', '0.27.0-RC1']
+        scala: ['2.11.12', '2.12.12', '2.13.3', '3.0.0-M3']
```

Comment:
Make sure to update github configuration to include the new check, and remove the old one.